package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "attractions",
-  "version": "0.0.0",
+  "version": "1.0.0-alpha",
```

Comment:
Is the `-alpha` prefix semver compliant?

Reply:
That's a suffix, and [yes](https://semver.org/)